lib/ui/survey_question/survey_question_item.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:flutter/material.dart';
+import 'package:survey_flutter_ic/model/survey_answer_model.dart';
+import 'package:survey_flutter_ic/model/survey_question_model.dart';
+import 'package:survey_flutter_ic/theme/dimens.dart';
+import 'package:survey_flutter_ic/ui/survey_question/answer/answer_dropdown.dart';
+
+class SurveyQuestionItem extends StatelessWidget {
+  final SurveyQuestionModel surveyQuestion;
+  final Function(SurveyAnswerModel) onDropdownSelect;
+
+  const SurveyQuestionItem({
+    super.key,
+    required this.surveyQuestion,
+    required this.onDropdownSelect,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return _buildPageItem(surveyQuestion);
+  }
+
+  Widget _buildPageItem(SurveyQuestionModel surveyQuestion) {
+    return Column(
+      children: [
+        _buildQuestionLabel(surveyQuestion.text),
+        Expanded(
+          // TODO: Handle widget to match with display type
+          child: AnswerDropdown(
+            answers: surveyQuestion.answers,
+            onSelect: (result) => onDropdownSelect.call(result),
+          ),
```

Comment:
Because we have many display types so I think we should extract a new view to handle inside `survey_answer_item`

Reply:
Noted @hoangnguyen92dn, I will consider `extracting` them on my current ticket [here](https://github.com/users/Wadeewee/projects/1/views/1?pane=issue&itemId=27222687) 🙇‍♀️

Reply:
@Wadeewee I can't see the project board. If you already address an issue for this so that's fine 👌